package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stardog",
-  "version": "1.4.0",
+  "version": "2.0.0",
   "description": "Stardog JavaScript Framework for node.js and the browser - Develop apps using the Stardog RDF Database & JS.",
```

Comment:
Marking is almost certainly going to want to change this from "Stardog RDF Database"